examples/js/controls/OrbitControls.js

```diff
@@ -366,38 +389,15 @@ THREE.OrbitControls = function ( object, domElement ) {
 
 	}();
 
-	function dollyIn( dollyScale ) {
+	function dolly( dollyScale, zoomOut ) {
```

Comment:
I'm not liking the zoomOut flag. Can you rewrite this using a signed quantity?

Also, maybe the first arg is a distance, not a scale?

Reply:
I was suggesting refactoring the code to remove the second argument -- not renaming it.

Shouldn't you be dollying a signed distance?

Reply:
Right, that's better with just one argument. I don't know about that argument being a scale or a distance. It definitely is a distance with `PerspectiveCamera` so I renamed the variable `scaleDelta` to `distanceDelta`, but with `OrthographicCamera` it just changes the zoom. So isn't the word 'scale' more appropriate (and actually 'dolly' misleading) in an isometric view ? Anyway I renamed it `dollyDistance` too for now.

Reply:
Yes, this is sort of a mess... Create the most elegant code you can, with a live demo, and we can have @mrdoob give it a look.

Reply:
`dolly ( dollyDistance )` is now `assignZoom ( zoomValue )`.